pkg/cluster/resources.go

```diff
@@ -154,6 +154,34 @@ func CreateNamespace(ctx context.Context, cli client.Client, namespace string, m
 	return desiredNamespace, client.IgnoreAlreadyExists(createErr)
 }
 
+// ProcessAllNamespace execute func() on all active namespaces in the cluster with 500 as chunksize.
+func ProcessAllNamespace(ctx context.Context, cli client.Client, processFunc func(*corev1.Namespace) error) error {
+	namespaces := &corev1.NamespaceList{}
+	paginateListOption := &client.ListOptions{
+		Limit: 500,
+	}
+
+	for { // for the case we have thousands of NS in the cluster
```

Comment:
If we really need to comment maybe this:
```suggestion
	for { // loop over all paged results
```